sympy/utilities/lambdify.py

```diff
@@ -450,6 +452,15 @@ def wrapper(*argsx, **kwargsx):
                     "Expression:\n\n{expr}").format(sig=sig, expr=expr_str)
     return func
 
+def print_dummify(expr):
+    valid_identifier_regex = re.compile('^[^\d\W]\w*\Z', re.UNICODE)
+    result = valid_identifier_regex.match(str(expr))
+    if result is None:
+        dummify = True
+    else:
+        dummify = False
+    return dummify
+
```

Comment:
I know it would be simple, but for understanding the code base purposes it would be difficult. When any other person would look at the added code he/she should be able to easily understand what's going on.